.changeset/fair-ducks-hope.md

```diff
@@ -0,0 +1,5 @@
+---
+'@feature-sliced/steiger-plugin': minor
```

Comment:
I think this qualifies more as a patch change, since it's not adding new features

Reply:
Indeed, you’re right